.github/workflows/tests-dbstorage.yml

```diff
@@ -61,7 +65,7 @@ jobs:
     - name: Setup cache
       uses: actions/cache@v2
       env:
-        cache-name: test-rdbstorage
+        cache-name: test-dbstorage
```

Comment:
Since Optuna doesn't have `DBStorage`, how about putting emphasis on `test with server`? 
```suggestion
        cache-name: test-storage-with-server
```

Reply:
Ok, I will fix it.

Reply:
Thank you!

Reply:
Fixed.

Reply:
Thanks!